Mathlib/Topology/TietzeExtension.lean

```diff
@@ -394,29 +394,29 @@ theorem exists_extension_forall_exists_le_ge_of_closedEmbedding [Nonempty X] (f
     intro x
     simp [dg0 (Or.inl <| mem_range_self _), ← hgf]
   refine' ⟨g - dg, fun y => _, funext hgf⟩
-  · have hyb : (g - dg) y < b := by
-      rcases (hgb y).eq_or_lt with (rfl | hlt)
-      · refine' (sub_lt_self_iff _).2 _
-        calc
-          0 < g y - c := sub_pos.2 hcb
-          _ = dg y := (dgb rfl).symm
-      · exact ((sub_le_self_iff _).2 (dgmem _).1).trans_lt hlt
-    rcases hb.exists_between hyb with ⟨_, ⟨xu, rfl⟩, hyxu, _⟩
-    cases' lt_or_le c (g y) with hc hc
-    · rcases em (a ∈ range f) with (⟨x, rfl⟩ | _)
-      · refine' ⟨x, xu, _, hyxu.le⟩
+  have hyb : (g - dg) y < b := by
```

Comment:
This block has to be manually checked.

Reply:
Did so: it's just github's diff being confused. Reviewing with whitespace hidden shows this is just un-cdoting.